js/interface.js

```diff
@@ -1832,11 +1840,21 @@ var DynamicLists = (function() {
       data.columnLabel = data.column === 'none'
         ? '(Field)'
         : data.column;
+      data.logicLabel = logicMap[data.logic]
```

Comment:
Where is `logicLabel` used? I thought we removed `logicMap`?

Reply:
@tonytlwu sorry, that point was missed after rebase the branch